lib/modules/versioning/distro.ts

```diff
@@ -23,6 +25,9 @@ export class DistroInfo {
     string,
     DistroInfoRecordWithVersion
   >();
+
+  private readonly _sortedInfo = new Array<DistroInfoRecordWithVersion>();
```

Comment:
please do refactoring in separate PR.

Reply:
I could do that no problem, 
but its not a refactor but rather an added functionality for the rolling releases.

Reply:
yes, please do in separate PR.